lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -281,59 +281,103 @@ module Vmpooler
         },
         'windows2003-64' => {
           'platform' => 'windows-2003-64',
-          'template' => 'win-2003-x86_64'
+          'template' => 'win-2003-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2003-6432' => {
+          'platform' => 'windows-2003-64',
+          'template' => 'win-2003-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows2003r2-32' => {
           'platform' => 'windows-2003r2-32',
-          'template' => 'win-2003r2-i386'
+          'template' => 'win-2003r2-i386',
+          'ruby_arch' => 'x86'
         },
         'windows2003r2-64' => {
           'platform' => 'windows-2003r2-64',
           'template' => 'win-2003r2-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2003r2-6432' => {
+          'platform' => 'windows-2003r2-64',
+          'template' => 'win-2003r2-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows2008-64' => {
           'platform' => 'windows-2008-64',
-          'template' => 'win-2008-x86_64'
+          'template' => 'win-2008-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2008-6432' => {
+          'platform' => 'windows-2008-64',
+          'template' => 'win-2008-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows2008r2-64' => {
           'platform' => 'windows-2008r2-64',
-          'template' => 'win-2008r2-x86_64'
+          'template' => 'win-2008r2-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2008r2-6432' => {
+          'platform' => 'windows-2008r2-64',
+          'template' => 'win-2008r2-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows2012-64' => {
           'platform' => 'windows-2012-64',
-          'template' => 'win-2012-x86_64'
+          'template' => 'win-2012-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2012-6432' => {
+          'platform' => 'windows-2012-64',
+          'template' => 'win-2012-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows2012r2-64' => {
           'platform' => 'windows-2012r2-64',
-          'template' => 'win-2012r2-x86_64'
+          'template' => 'win-2012r2-x86_64',
+          'ruby_arch' => 'x64'
+        },
+        'windows2012r2-6432' => {
+          'platform' => 'windows-2012r2-64',
+          'template' => 'win-2012r2-x86_64',
+          'ruby_arch' => 'x86'
         },
         'windows7-64' => {
           'platform' => 'windows-7-64',
-          'template' => 'win-7-x86_64'
+          'template' => 'win-7-x86_64',
+          'ruby_arch' => 'x64'
```

Comment:
Cool, I mean the data is syntactically correct, as long as it's the correct data. I've been pinged on what looks like a re-evaluation of our Windows 32 bit testing matrices elsewhere, so I wanted to broach the subject.

I'm also fine, if getting a definitive answer is going to take awhile, to merge this and update it later if needed.